modules/creating-manifest-file-multiple-interfaces-br-ex.adoc

```diff
@@ -0,0 +1,213 @@
+// Module included in the following assemblies:
+//
+// * support/troubleshooting/troubleshooting-network-issues.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="creating-manifest-file-multiple-interfaces-br-ex_{context}"]
+= Creating an additional OVN-Kubernetes `br-ex` bridge network
+
+You can create an additional `br-ex` bridge network, `br-ex1`, that has a different configuration to the default `br-ex` bridge. You must define the additional bridge in an interface definition of a machine configuration manifest file. The Machine Config Operator uses the manifest to create a new file at `/etc/ovnk/extra_bridge` on the host. The new file includes the name of the bond interface that hosts the additional `br-ex` bridge network.
+
+After you create and edit the manifest file, the Machine Config Operator completes tasks in the following order:
```

Comment:
If this is an ordered list, the items should be numbered rather than bulleted.

Reply:
Confirmed with SME:

> to answer these can do numbered list, and we can skip restart step, machine config apply will restart hosts, but to verify the MAC address of BREX will have the same MAC as the iface that it is using for the connection- debugging into the node and checking ip a will allow us to compare.